mailet/standard/src/test/java/org/apache/james/transport/matchers/HostIsLocalTest.java

```diff
@@ -46,7 +46,7 @@
 
     @Before
     public void setUp() throws Exception {
-        mockedMail = new FakeMail();
+        mockedMail = FakeMail.builder().build();
```

Comment:
Well. Yes, it can and I found another issue with that builder, will commit soon.